docs/intro/overview.md

```diff
@@ -0,0 +1,56 @@
+---
+sidebarLabel: Overview
+title: "What is Solana?"
+sidebarSortOrder: 1
+description:
+  "Solana is the high-performance blockchain designed for mass adoption. Learn
+  why Solana is the top choice for developers looking to build scalable
+  blockchain applications."
+altRoutes:
+  - /docs/intro/history
+  - /docs/intro
+---
+
+Solana is a blockchain built for mass adoption. It's a high performance network
+that is utilized for a range of use cases, including finance, payments, and
+gaming. Solana operates as a single global state machine, and is open,
+interoperable, and decentralized.
+
+## Why Solana?
+
+Designed to scale, Solana is purpose built for blockchain applications to reach
+millions of users. Instead of worrying about optimizing for the blockchain
+layer, developers can focus on building their application to reach product
+market fit. Not only can it scale now for blockchain application needs, but the
+network continues to be optimized with user experience in mind.
+
+Building the best user experience in an application is a top priority for
+developers. With blockchains, the user experience is often limited by the
+underlying technology, causing slow response times and high fees. Solana's low
+fees and 400ms confirmation times enable developers to build applications that
+are user friendly and accessible to everyone.
```

Comment:
```suggestion
## Why Solana?

Solana's low fees and 400ms confirmation times enable developers to build applications that are responsive and scale to millions of users. Instead of worrying about optimizing for blockchains with slow response times and high fees, Solana allows developers to focus on building their application to reach product
market fit. 
```

Reply:
I think we can say this with less words, so people can see the amazing list of features in the next section.